source/module_base/memory.cpp

```diff
@@ -234,10 +234,10 @@ double Memory::record_gpu
 {
 	if(!Memory::init_flag_gpu)
 	{
-		name_gpu = new std::string[n_memory];
-		class_name_gpu = new std::string[n_memory];
-		consume_gpu = new double[n_memory];
-		for(int i=0;i<n_memory;i++)
+		name_gpu = new std::string[n_memory + 2];
```

Comment:
why not use std::vector instead?

Reply:
There is no need to use `n_memory+2` here. I just forget to change back my code for debugging 😭